xml/System.Linq/Enumerable.xml

```diff
@@ -12608,12 +12608,12 @@ Only unique elements are returned.
         <Parameter Name="second" Type="System.Collections.Generic.IEnumerable&lt;TSecond&gt;" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <typeparam name="TFirst">To be added.</typeparam>
-        <typeparam name="TSecond">To be added.</typeparam>
-        <param name="first">To be added.</param>
-        <param name="second">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="TFirst">The type of the elements of the first input sequence.</typeparam>
+        <typeparam name="TSecond">The type of the elements of the second input sequence.</typeparam>
+        <param name="first">The first sequence to merge.</param>
+        <param name="second">The second sequence to merge.</param>
+        <summary>Applies a specified function to the corresponding elements of two sequences, producing a sequence of value types with elements from both sequences.</summary>
```

Comment:
There's no "specified function".  This overload is different from the existing one.  The existing one takes an item from each sequence and applies a function to both inputs to determine what value to yield.  In contrast, this overload takes the two inputs and just yields a tuple of those two values.

Reply:
So would this be correct @stephentoub?
  
```suggestion
        <summary>Produces a sequence of value types with elements from the two specified sequences.</summary>
```